packages/components-native/package.json

```diff
@@ -21,10 +21,11 @@
     "build:clean": "rm -rf ./dist"
   },
   "dependencies": {
+    "lodash.identity": "^3.0.0",
     "@jobber/design": "^0.41.0",
     "lodash.chunk": "^4.2.0",
     "react-hook-form": "^7.30.0",
-    "react-intl": "^6.4.2",
+    "react-intl": "^4.7.5",
```

Comment:
Could we consider bumping Jobber Mobile up to 6.4.2?

Reply:
I can take a look at their changelog and see if there's anything that would affect us. But considering it's 2 major version bumps, I'm hesitant to bump it on jobber-mobile instead of matching the version here in Atlantis. The package is being used more extensively in jobber-mobile and we would have to QA it very well to make sure we're not breaking anything.

Reply:
There's an open PR to upgrade it: https://github.com/GetJobber/jobber-mobile/pull/7348